packages/vkui/src/components/InfoRow/InfoRow.tsx

```diff
@@ -19,9 +20,12 @@ export const InfoRow = ({ header, children, className, ...restProps }: InfoRowPr
     weight="3"
   >
     {hasReactNode(header) && (
-      <Subhead Component="span" className={styles['InfoRow__header']}>
-        {header}
-      </Subhead>
+      <>
+        <Subhead Component="strong" className={styles['InfoRow__header']}>
+          {header}
+        </Subhead>
+        <VisuallyHidden>&nbsp;</VisuallyHidden>
+      </>
```

Comment:
А вот так можно?

```suggestion
      <Subhead Component="strong" className={styles['InfoRow__header']}>
        {header}<VisuallyHidden>&nbsp;</VisuallyHidden>
      </Subhead>
```

чтобы без фрагмент

**UPD**

Судя по `TabsItem.tsx` можно. Поправим без фрагмент тогда?

Reply:
➕